bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/ParadoxCommunicatorFactory.java

```diff
@@ -60,8 +60,8 @@ public IParadoxCommunicator createCommunicator(PanelType panelType)
             throws UnknownHostException, IOException, ParadoxException {
         switch (panelType) {
             case EVO48:
+            case EVO96:
             case EVO192:
-            case EVOHD:

```

Comment:
Did you remove the EVOHD intentionally?

Reply:
Yes. We figured that the packets used for EVOHD during logon are differeent, so I removed it and I've added EVO96 if I'm not wrong... Basically expectations are that EVO48/96/192 use same protocol. I expected that HD is also using the same but some people tried it and it cannot even pass the logon sequence. Probably if this needs to be supported someone with the necessary skills needs to reverse engineer the protocol and implement new communicator for it.